internal/storage/db/internal/db.go

```diff
@@ -603,16 +608,21 @@ func (s *dbStorage) Delete(ctx context.Context, ids ...namer.ModuleID) error {
 	return nil
 }
 
-func (s *dbStorage) ListPolicyIDs(ctx context.Context) ([]string, error) {
+func (s *dbStorage) ListPolicyIDs(ctx context.Context, includeDisabled bool) ([]string, error) {
 	var policyCoords []namer.PolicyCoords
+	var whereExprs []exp.Expression
+	if !includeDisabled {
+		whereExprs = append(whereExprs, goqu.C(PolicyTblDisabledCol).Neq(goqu.V(true)))
+	}
```

Comment:
I think this can be simplified. Just always add a WHERE clause to the query.

```suggestion
goqu.C(PolicyTblDisabledCol).Eq(goqu.V(includeDisabled)))
```

Reply:
Then we will have only disabled policies or enabled policies depending on the `includeDisabled` parameter. But, we actually want to have all policies when `includeDisabled == true`.

Reply:
Good point! I completely messed that one up 🤦🏽